config.ru

```diff
@@ -0,0 +1,2 @@
+require './i-scream-arcade-microservice-app'
```

Comment:
**When to use a config.ru?** A config.ru file is recommended if:

_You want to deploy with a different Rack handler (Passenger, Unicorn, Heroku, …)._
You want to use more than one subclass of Sinatra::Base.
You want to use Sinatra only for middleware, and not as an endpoint.